memlab/memoryLeakTest.js

```diff
@@ -0,0 +1,13 @@
+function url() {
+  return 'http://localhost:3000/';
```

Comment:
It will be better to use the environment variable for this link from the .env file. You can check [this library](https://www.npmjs.com/package/dotenv)